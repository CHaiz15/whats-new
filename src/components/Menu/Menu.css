.Menu {
  display: flex;
  justify-content: space-between;
  width: 100%;
  margin: 5px;
}

.src-btn {
  width: 90%;
  height: 50px;
}
