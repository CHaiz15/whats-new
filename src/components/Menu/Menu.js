import React from 'react';
import './Menu.css'

const Menu = ({updateSource}) => {
    return (
      <div className='Menu'>
        <button className='src-btn' name='local' onClick={updateSource}>Local News</button>
        <button className='src-btn' name='technology' onClick={updateSource}>Technology</button>
        <button className='src-btn' name='entertainment' onClick={updateSource}>Entertainment</button>
        <button className='src-btn' name='science' onClick={updateSource}>Science</button>
        <button className='src-btn' name='health' onClick={updateSource}>Health</button>
      </div>
    )
  }


export default Menu;
