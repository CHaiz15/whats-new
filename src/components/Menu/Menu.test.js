import React from 'react'
import { shallow } from 'enzyme'
import Menu from './Menu.js'

describe('Menu', () => {
  it('renders an indication Menu', () => {
    const wrapper = shallow(<Menu
      updateSource={jest.fn()}
    />);

    expect(wrapper).toMatchSnapshot();
  })
})
