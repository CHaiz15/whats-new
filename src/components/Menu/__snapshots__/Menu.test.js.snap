// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Menu renders an indication of Menu 1`] = `
<div
  className="Menu"
>
  <button
    className="src-btn"
    name="local"
    onClick={[MockFunction]}
  >
    Local News
  </button>
  <button
    className="src-btn"
    name="technology"
    onClick={[MockFunction]}
  >
    Technology
  </button>
  <button
    className="src-btn"
    name="entertainment"
    onClick={[MockFunction]}
  >
    Entertainment
  </button>
  <button
    className="src-btn"
    name="science"
    onClick={[MockFunction]}
  >
    Science
  </button>
  <button
    className="src-btn"
    name="health"
    onClick={[MockFunction]}
  >
    Health
  </button>
</div>
`;
